.travis.yml

```diff
@@ -2,6 +2,7 @@ language: python
 env:
   - PYTHON_VERSION=2.7
 before_install:
+  - redis-server --version
```

Comment:
Forgot to mention that this is not blocking, meaning that if other @biocore/qiita-dev thinks that this is ok to leave; then he can merge...